lib/l10n/app_ja.arb

```diff
@@ -0,0 +1,15 @@
+{
+  "chooseAccountPageTitle": "アカウントを選択",
```

Comment:
What does the process look like when adding a new translated string?

I imagine in particular it includes:
* add something in `app_en.arb`
* ?? rerun code generation
* use the resulting getter or method in app code

Is it necessary to also add something to all the other .arb files, or will things fall back smoothly until we have a translation?

What's the most convenient way to get the generated code to refresh after adding to a .arb file?

Reply:
`flutter gen-l10n` will recreate the bindings and my IDE picks it up after that straightaway. The generate command also runs automatically as part of `flutter run` so builds will show the most up to date strings. I only need to run `flutter gen-l10n` if I need my IDE to autocomplete a newly added string.

I've tested with missing strings and the situation there is it will use a fallback string. The generate command (or run) will then output a message alerting you that strings are missing:

```
bash-3.2$ fvm flutter gen-l10n
Because l10n.yaml exists, the options defined there will be used instead.
To use the command line arguments, delete the l10n.yaml file in the Flutter project.


"ja": 1 untranslated message(s).
To see a detailed report, use the untranslated-messages-file
option in the l10n.yaml file:
untranslated-messages-file: desiredFileName.txt
<other option>: <other selection>


This will generate a JSON format file containing all messages that
need to be translated.
```

It looks like we can get the untranslated messages into a JSON file using the `untranslated-messages-file` option, which I presume we somehow upload to the translation management service. I didn't add this setting or look into it as I believe it was out of scope for this issue.